sdk/src/types/block/output/rent.rs

```diff
@@ -140,3 +148,51 @@ impl<T: Rent, const N: usize> Rent for [T; N] {
         self.iter().map(|elem| elem.weighted_bytes(config)).sum()
     }
 }
+
+pub struct MinimumStorageDepositBasicOutput {
+    config: RentStructure,
+    token_supply: u64,
+    builder: BasicOutputBuilder,
+}
+
+impl MinimumStorageDepositBasicOutput {
+    pub fn new(config: RentStructure, token_supply: u64) -> Self {
+        Self {
+            config,
+            token_supply,
+            builder: BasicOutputBuilder::new_with_amount(Output::AMOUNT_MIN).add_unlock_condition(
```

Comment:
Isn't there a `new_with_min_storage_deposit` variant?

Reply:
There is, but in this case the result is the same, but `new_with_min_storage_deposit` induces a clone of the output